docs/source/notebook.rst

```diff
@@ -257,53 +265,51 @@ supports a `large subset <mathjax_tex>`_ of LaTeX functionality
 .. _mathjax_tex: http://docs.mathjax.org/en/latest/tex.html
 
 Standard mathematics environments defined by LaTeX and AMS-LaTeX (the
-`amsmath` package) also work, such as
-``\begin{equation}...\end{equation}``, and ``\begin{align}...\end{align}``.
-New LaTeX macros may be defined using standard methods,
+`amsmath` package), such as begin{equation}...\end{equation}``, and ``\begin{align}...\end{align}``
+will also be correctly displayed.
+You can define new LaTeX macros using standard methods,
 such as ``\newcommand``, by placing them anywhere *between math delimiters* in
 a Markdown cell. These definitions are then available throughout the rest of
-the IPython session.
+the notebook session.
 
 .. seealso::
 
     `Markdown Cells`_ example notebook
 
-Raw cells
-~~~~~~~~~
-
-*Raw* cells provide a place in which you can write *output* directly.
-Raw cells are not evaluated by the notebook.
-When passed through nbconvert_, raw cells arrive in the
-destination format unmodified. For example, this allows you to type full LaTeX
-into a raw cell, which will only be rendered by LaTeX after conversion by
-nbconvert.
-
-Heading cells
-~~~~~~~~~~~~~
+Headings
```

Comment:
Strike entire section on headings. 4.1 no longer makes a distinction between Markdown cells and headings. They are all Markdown cells now.